lib/Gateway.js

```diff
@@ -11,26 +13,20 @@ const hassCfg = reqlib('/hass/configurations.js')
 const hassDevices = reqlib('/hass/devices.js')
 const version = reqlib('package.json').version
 
-debug.color = 2
-
-try {
-  const storeDir = reqlib('config/app.js').storeDir
-  const customDevices = reqlib(storeDir + '/customDevices')
-  let i = 0
-  for (const d in customDevices) {
-    hassDevices[d] = customDevices[d]
-    i++
-  }
+const STORE_DIR = reqlib('config/app.js').storeDir
+const CUSTOM_DEVICES = 'customDevices.json'
```

Comment:
Import `lib/utils` and use` utils.joinPath` because it keep sure the path is correct when using `pkg` files.
Remove STORE_DIR var and use just the:

```js
const CUSTOM_DEVICES = utils.joinPath(true, reqlib('config/app.js').storeDir, 'customDevices')
```

And add `js|json` extensions on the methods